app/src/main.ts

```diff
@@ -1,14 +1,22 @@
 import './assets/main.css'
-
+import { WagmiPlugin } from '@wagmi/vue'
+import { config } from './wagmi.config'
+import { QueryClient, VueQueryPlugin } from '@tanstack/vue-query'
 import { createApp } from 'vue'
 import { createPinia } from 'pinia'
-
 import App from './App.vue'
 import router from './router'
 
-const app = createApp(App)
+export function setupApp() {
```

Comment:
Do you need to export setup app here ?

Reply:
@dasarathg68

Reply:
I can't test main.ts directly so I have created a function setupApp that uses WagmiPlugin, router, pinia, VueQueryPlugin and exported it. 
Below I have mounted it by calling the setupApp() function. 
Like this:
```typescript
setupApp().mount('#app')
```